index.html

```diff
@@ -99,6 +105,13 @@ <h2>Hauptleitung</h2>
         padding: 0.3rem 0.8rem;
     }
 
+    main img.ellipsis {
+        border: 1px solid white;
+        border-radius: 9999px;
+        height: 7rem;
+        margin: 5px;
+    }
```

Comment:
I'm not sure it's wise to have such large radii for the sponsors' logos. Messing with other people's logos is tricky, they may not want it. I like how you attempt to integrate it with the design, but I'd suggest not doing it as much. And a white border for white background images is hardly needed.

I would put a small radius on the corners, otherwise it may look a bit too harsh, but just a small one. What do you think?

Reply:
The Logos are fully visible, so i am not really concerned that this could be a problem. I did center the images in the logo and added Margins, so that it is fully visible in the Buttons. I wouldn't change it.